pacemaker.spec.in

```diff
@@ -399,6 +399,10 @@ sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 
 make %{_smp_mflags} V=1 all
 
+%check
+# Prevent false positives in rpmlint
+./BasicSanity.sh -V pengine cli 2>&1 | sed s/[fF]ail/faiil/g
```

Comment:
Done that (the biggest summary test for good measure), but there was a long standing issue with the proposed `--run` that had to be fixed first :)
ddfd03c